### DISCUSSION TITLE
Whether multi views can based on a shared data to add its own data?

### DISCUSSION THREAD

**Discussion by liaodalin19903** [cytoscape/cytoscape.js#3297] [Q&A]
Cytoscape can generate `View 1` based on `Data 1`. Then, more custom data for the view can be added to `View 1` (so the current data of `View 1` is: `Data 1` + `Data View 1`). After that, more data can also be added to `View 2` based on `Data 1` (so the current data on `View 2` is: `Data 1` + `Data View 2`).

1. If we modify Data 1 in `View 1`, then `Data 1` in `View 2` will change accordingly (because the same base data is used). Can this be achieved in Cytoscape?

2. If not, is it possible to create `View 1` and `View 2` for `Data 1`, and then have` View 1` display only specific filtered `Data 1-1` and `View 2` display only specific filtered `Data 1-2`?

**Comment by mikekucera** [comment]
Please post questions about Cytoscape desktop to the cytoscape-helpdesk google group: https://groups.google.com/g/cytoscape-helpdesk
